modules/sdk-coin-avaxp/test/resources/avaxp.ts

```diff
@@ -73,14 +73,18 @@ export const INVALID_RAW_TRANSACTION = 'This is an invalid raw transaction';
 
 export const INVALID_ADDRESS = 'INVALID-ADDRESS';
 
-export const VALID_ADDRESS = 'avax14hhmsl9um7e9xjwqgvkf9rswq0y0zk9te8plfv';
+export const VALID_ADDRESS = 'P-avax14hhmsl9um7e9xjwqgvkf9rswq0y0zk9te8plfv';
 
-export const VALID_BLOCK_ID = '0xe593ee0c4b2a53a770f98720ebcb676d8e55102826c4e4ecc5112da9cf737d57';
+export const VALID_BLOCK_ID = 'nQna7h6aAe2kN26eViH3qWU9yProgc4abr1SzKYwjX6HA7v6d';
```

Comment:
@jicli Could you confirm if block id could be hex?

Reply:
i dont think so, it's standard to use scp256 hash